### PR TITLE
fix(TileView): not showing all participants.

### PR DESCRIPTION
**Pull request by hristoterezov**

### CODE REVIEW

File: react/features/filmstrip/actions.native.js [hunk 1]
```diff
@@ -3,7 +3,8 @@
 import {
     SET_FILMSTRIP_ENABLED,
     SET_FILMSTRIP_VISIBLE,
-    SET_TILE_VIEW_DIMENSIONS
+    SET_TILE_VIEW_DIMENSIONS,
+    SET_REMOTE_PARTICIPANTS
```
Comment (jallamsetty1) [inline react/features/filmstrip/actions.native.js:7]: nit: alpha sort.

File: react/features/filmstrip/actions.native.js [hunk 2]
```diff
@@ -60,3 +61,20 @@ export function setTileViewDimensions({ thumbnailSize }: Object) {
...
+ * Sets the list of the reordered remote participants based on which the visible participants in the filmstrip will be
+ * determined.
+ *
+ * @param {Array<string>} participants - The list of the remote participant endpoint IDs.
+ * @returns {{
+        type: SET_REMOTE_PARTICIPANTS,
+        participants: Array<string>
+    }}
+ */
+export function setRemoteParticipants(participants: Array<string>) {
```
Comment (jallamsetty1) [inline react/features/filmstrip/actions.native.js:75]: nit: alpha sort, should be inserted before `setTileViewDimensions` function.

Comment (jallamsetty1) [reply react/features/filmstrip/actions.native.js:75]: Also, don't we want to move `setRemoteParticipants` to a common file actions.any.js. ?

**Review by jallamsetty1 — COMMENTED** [review]
LGTM Hristo, left 1 minor comment and a question.

File: react/features/filmstrip/actions.native.js (hunk 2 above)
Comment (hristoterezov) [reply react/features/filmstrip/actions.native.js:75]: will move it to a common file! Thanks for the idea!

File: react/features/filmstrip/actions.web.js [hunk 3]
```diff
@@ -26,22 +25,7 @@ import {
...
-        participants: Array<string>
-    }}
- */
-export function setRemoteParticipants(participants: Array<string>) {
-    return {
-        type: SET_REMOTE_PARTICIPANTS,
-        participants
-    };
-}
+export * from './actions.any';
```
Comment (jallamsetty1) [inline react/features/filmstrip/actions.web.js:28]: Now that we have actions.any.js, do we need this [line](https://github.com/jitsi/jitsi-meet/blob/master/react/features/filmstrip/actions.web.js#L196) still ?

Comment (hristoterezov) [reply react/features/filmstrip/actions.web.js:28]: Yes. Without it we won't export the common actions in the web version.

Comment (jallamsetty1) [reply react/features/filmstrip/actions.web.js:28]: I was referring to https://github.com/jitsi/jitsi-meet/blob/master/react/features/filmstrip/actions.web.js#L196 where the actions.native are re-exported from actions.web.

Comment (hristoterezov) [reply react/features/filmstrip/actions.web.js:28]: I saw the problem. will fix it

**Review by jallamsetty1 — APPROVED** [review]

**Pull request merged**